app/assets/javascripts/index.js.erb

```diff
@@ -140,27 +140,45 @@ class BirdMap {
 		marker.birdId = bird.subId;
 
 		marker.on('click', () => {
-			$('.overlay')
-				.html(`
-					<div class="expand-icon"></div>
-					<div class="content">
-						<div class="bird-name">${bird.name.com}</div>
-						<div class="science-name">${bird.name.sci}</div>
-						<div class="bird-img-wrapper">
-							<img class="bird-img" src="/duck.png">
-						</div>
-						<div class="location-wrapper">
-							<div class="location">Loc</div>
-							<div class="separator"><i class="fa-solid fa-circle"></i></div>
-							<div class="distance">${Unit.m2mi(marker.getLatLng().distanceTo(this.position)).toFixed(2)} mi</div>
-						</div>
-						<div class="description-wrapper">
-							<div class="description-header">Description:</div>
-							<div class="description-content">No Description Available</div>
-					</div>
-					</div>
-				`)
+			const request = {
+			url: '/ebird/bird',
+			method: 'GET',
+			timeout: 0,
+			data: {
+				sci: bird.name.sci,
+				com: bird.name.com
+				}
+			};

```

Comment:
The request should be in the same format located in the class `Ebird` on this JS file. you can convert the `marker.on('click')` to an asynchronous function with `marker.on('click', async () => {})` and use `await` to wait for responses